src/docs/sdk/client-reports.mdx

```diff
@@ -2,6 +2,13 @@
 title: Client Reports
 sidebar_order: 20
 ---
+## Scope and Intent
```

Comment:
_nit_: Maybe move this section after the two paragraphs that already existed on the top of the document. They are a better introduction (and a callout for the Relay bug), before going into the scope and intent.

Reply:
@jan-auer better?